config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  get 'users/homepage' , 'users#homepage'
+  get 'home/homepage'
   devise_for :users
-  resources :movies
-  root to: 'movies#index'
+  resources :movies, except: [:index]
+  root to: 'home#homepage'
```

Comment:
Lets create `dashboard` action instead of `homepage`, so it'll be `home#dashboard`